src/components/InventoryTable/ConversionPopover/ConversionPopover.js

```diff
@@ -18,15 +18,15 @@ const ConversionPopover = () => {
       headerContent={'Convert this system to RHEL'}
       bodyContent={
         <TextContent>
-          <Text style={{ maxWidth: '380px' }}>
+          <Text style={{ maxWidth: '380px', fontSize: '14px' }}>
```

Comment:
Nitpick: Can these [font sizes ](https://www.patternfly.org/utility-classes/text/#font-size)from PF be applied?